support-frontend/app/services/stepfunctions/SupportWorkersClient.scala

```diff
@@ -119,6 +120,7 @@ class SupportWorkersClient(
     requestId: UUID,
     promoCode: Option[PromoCode] = None
   ): EitherT[Future, SupportWorkersError, StatusResponse] = {
+    SafeLogger.info(s"$requestId: debug info ${request.body.debugInfo}")
```

Comment:
I think we should probably remove this logging. It will be pretty noisy and this is not where we need that information really.

Reply:
Yes agree the whole point of thee exercise is to collect all the info on the support worker. feel free to remove that as I'm on the bus now!

Reply:
this logging is back as sending to support workers causes a length limit to be exceeded